timesketch/frontend-ng/src/store.js

```diff
@@ -255,5 +263,98 @@ export default new Vuex.Store({
         console.error(e)
       })
     },
+    updateAnalyzerList(context, sketchId) {
+      if (!sketchId) {
+        sketchId = context.state.sketch.id
+      }
+      ApiClient.getAnalyzers(sketchId).then((response) => {
+        let analyzerList = {}
+        if (response.data !== undefined) {
+          response.data.forEach((analyzer) => {
+            analyzerList[analyzer.name] = analyzer
+          })
+        }
+        context.commit('SET_ANALYZERS', analyzerList)
+      }).catch((e) => {
+        console.log(e)
+      })
+    },
+    updateAnalyzerResults(context, sketchId) {
+      if (!sketchId) {
+        sketchId = context.state.sketch.id
+      }
+      let analyses = []
+      context.state.sketch.timelines.forEach(timeline => {
+        ApiClient.getSketchTimelineAnalysis(sketchId, timeline.id)
```

Comment:
I opt to move this logic to the component instead, and use a computed property; We can sync up pn this in person :)

Reply:
I moved it back into the component as part of the `mounted()` function. Since my logic is based on async functions, this does not work as a computed method (https://eslint.vuejs.org/rules/no-async-in-computed-properties.html).